src/utils/dates.ts

```diff
@@ -1,13 +1,4 @@
-import { DateTime } from 'luxon'
-
-declare global {
-  interface Window {
-    timezone?: {
-      system?: string
-      user?: string
-    }
-  }
-}
+import { dayjs } from './dayjs'
```

Comment:
why is this file needed?

Reply:
moved all datepicker components to separate folder with dates.ts -> utils.ts